docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -197,13 +197,15 @@ Just like in [Designer installation](./Installation.md#Basic environment variabl
-| http.interface                                            | string          | 0.0.0.0         | Interface on which REST API of scenarios will be exposed                                                                         |
-| http.port                                                 | int             | 8181            | Port on which REST API of scenarios will be exposed                                                                              | 
-| request-response.definitionMetadata.servers               | string          | [{"url": "./"}] | Configuration of exposed servers in scenario's OpenApi definition. When not configured, will be used server with ./ relative url | 
-| request-response.definitionMetadata.servers[].url         | string          |                 | Url of server in scenario's OpenApi definition                                                                                   | 
-| request-response.definitionMetadata.servers[].description | string          |                 | (Optional) description of server in scenario's OpenApi definition                                                                | 
+| Parameter                                                 | Type   | Default value   | Description                                                                                                                      |
+|-----------------------------------------------------------|--------|-----------------|----------------------------------------------------------------------------------------------------------------------------------|
+| http.interface                                            | string | 0.0.0.0         | Interface on which REST API of scenarios will be exposed                                                                         |
+| http.port                                                 | int    | 8181            | Port on which REST API of scenarios will be exposed                                                                              | 
+| request-response.definitionMetadata.servers               | string | [{"url": "./"}] | Configuration of exposed servers in scenario's OpenApi definition. When not configured, will be used server with ./ relative url | 
+| request-response.definitionMetadata.servers[].url         | string |                 | Url of server in scenario's OpenApi definition                                                                                   | 
+| request-response.definitionMetadata.servers[].description | string |                 | (Optional) description of server in scenario's OpenApi definition                                                                | 
+| request-response.basicAuthConfig.user                     | string |                 | (Optional) Basic auth user                                                                                                       | 
```

Comment:
What do you think about nesting in under `security` (so it will be `request-response.security.basicAuth.user`
), therefore when we introduce another type of auth in the future, we will not have to change basic-auth api?

Reply:
done